src/main/java/org/jenkinsci/plugins/badge/StatusImage.java

```diff
@@ -94,11 +96,11 @@
      */
     @Override
     public void generateResponse(StaplerRequest req, StaplerResponse rsp, Object node) throws IOException, ServletException {
-        String v = req.getHeader("If-None-Match");
+        /*String v = req.getHeader("If-None-Match");
```

Comment:
This piece of code can be re-enabled safely once this version stabilizes. However, we should document that this piece of code should be disabled whenever someone attempts to modify the code, otherwise, none of the changes would get reflected. I have left it commented out for now.